File: C00_objectCcallswift/C00_objectCcallswift/ViewController.m
//
//  ViewController.m
//  C00_objectCcallswift
//
//  Created by ucom Apple 13 on 2016/11/25.
//  Copyright © 2016年 GinaY. All rights reserved.
//

#import "ViewController.h"
#import "C00_objectCcallswift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    User *u = [User new];
    u.name = @"Gina" ;
    u.age = 23;
    [u showInfo];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 00_swiftObjC/00_swiftObjC/ViewController.swift
//
//  ViewController.swift
//  00_swiftObjC
//
//  Created by ucom Apple 13 on 2016/11/25.
//  Copyright © 2016年 GinaY. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let user = User()
        user.name = "Gina"
        user.age = 23
        user.showInfo()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

